README.md

```diff
@@ -858,6 +882,402 @@ summary](#compare-followers-databases-against-leader-view).
+
+The **CompareRaftMembers/compareFollowerDbs** tool compares two sets
+of Metadata leveldb databases, belonging to two different Metadata
+nodes, and outputs the differences to the file path given as the
+DIFF_OUTPUT_FILE environment variable.
+
+In this file, it outputs each key that differs as line-separated JSON
+entries, where each entry can be one of:
+
+- `[{ key, value }, null]`: this key is present on the follower #1 but
+  not on follower #2
+
+- `[null, { key, value }]`: this key is not present on follower #1
+  but is present on follower #2
+
+- `[{ key, value: "{value1}" }, { key, value: "{value2}" }]`: this key
+  has a different value between follower #1 and follower #2: "value1"
+  is the value seen on follower #1 and "value2" the value seen on
+  follower #2.
+
+## Usage
+
+```
+node compareFollowerDbs.js
+```
+
+## Mandatory environment variables
+
+* **DATABASES1**: space-separated list of databases of follower #1 to
+  compare against follower #2
+
+* **DATABASES2**: space-separated list of databases of follower #2 to
+  compare against follower #1
+
+* **DIFF_OUTPUT_FILE**: file path where diff output will be stored
+
+## Optional environment variables
+
+* **PARALLEL_SCANS**: number of databases to scan in parallel (default 4)
+
+* **EXCLUDE_FROM_CSEQS**: mapping of raft sessions to filter on, where
+  keys are raft session IDs and values are the cseq value for that
+  raft session. Filtering will be based on all oplog records more
+  recent than the given "cseq" for the raft session. Input diff
+  entries not belonging to one of the declared raft sessions are
+  discarded from the output. The value must be in the following JSON
+  format:
+
+  * `{"rsId":cseq[,"rsId":cseq...]}`
+
+  * Example: `{"1":1234,"4":4567,"6":6789}`
+
+  * This configuration would cause diff entries of which bucket/key
+    appear in one of the following to be discarded from the output:
+
+    * oplog of raft session 1 after cseq=1234
+
+    * or oplog of raft session 4 after cseq=4567
+
+    * or oplog of raft session 6 after cseq=6789
+
+    * or any other raft session's oplog at any cseq
+
+* **BUCKETD_HOSTPORT**: ip:port of bucketd endpoint, needed when
+  EXCLUDE_FROM_CSEQS is set
+
+## Scan Procedure with Unstable Leader
+
+**Note**: you may use this procedure when leaders are unstable,
+i.e. leader changes are frequent or susceptible to happen during the
+duration of the scan procedure. When leaders are stable, it can be
+more practical and efficient to use the [stable
+leader](#scan-procedure-with-stable-leader) procedure instead.
+
+In order to scan a set of raft sessions to detect discrepancies
+between raft members, you may use the following procedure involving
+the scan tool [compareFollowerDbs](#compare-two-followers-database-sets).
+
+The general idea is to compare sets of databases belonging to two
+different followers at a time.
+
+**Note**: Each repd node (including the current leader) has to be
```

Comment:
a link/reference to the stopping and starting commands could be helpful here

Reply:
The stop/start commands are fully included in the procedure itself. (e.g. https://github.com/scality/s3utils/pull/290/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R770), here the point is mostly to warn the user that repds will be stopped temporarily, which can have an impact on the platform. I could mention this fact though.